README.md

```diff
@@ -198,22 +194,16 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [Crystal-Maze](https://github.com/Demonstrandum/Crystal-Maze) - A* Path finding for PNG mazes
  * [crystal-mysql-crud-example](https://github.com/codenoid/crystal-mysql-crud-example) - Crystal MySQL CRUD example
  * [crystal-patterns](https://github.com/crystal-community/crystal-patterns) - Examples of GOF patters
- * [crystal_samples](https://github.com/tbpgr/crystal_samples) - Variety of Crystal samples
  * [crystalized_ruby](https://github.com/phoffer/crystalized_ruby) - Native Ruby extensions written in Crystal
- * [docker-kemal](https://github.com/ianblenke/docker-kemal) - An example Dockerized Crystal Kemal project
  * [exercism-crystal](https://github.com/exercism/crystal) - Exercism exercises
  * [jihantoro-cr-mysql](https://github.com/codenoid/jihantoro-cr-mysql) - Crystal MySQL from scratch sample app
  * [jihantoro.sd](https://github.com/codenoid/jihantoro.sd) - Crystal & Kemal version of Serdar Dogruyol blog
  * [kemal-chat](https://github.com/sdogruyol/kemal-chat) - Build realtime applications with Kemal and WebSocket
  * [kemal-heroku-example](https://github.com/cagataycali/kemal-heroku-example) - This repository shows, how you can publish your open source apps which powered kemal framework publish as heroku app in seconds
  * [kemal-mysql-blog](https://github.com/codenoid/kemal-mysql-blog) - Blog written with Crystal, Kemal and MySQL
  * [kemal-react-chat](https://github.com/f/kemal-react-chat) - Build Realtime Web applications with Kemal and React
- * [kemal-react-pg-chat](https://github.com/Angarsk8/chat-app-demo) - Chat application developed with Kemal, React, ES2015 and PostgreSQL
- * [kemal-ws-pg-todo-app](https://github.com/Angarsk8/kemal-ws-pg-todo-app) - Realtime Todo application developed with Kemal, Websockets, ES2015 and PostgreSQL
- * [kemal_elm_chat](https://github.com/kofno/kemal_elm_chat) - Simple chat server written with Kemal and Elm
  * [lattice-core-card-game](https://github.com/jasonl99/card_game) - A demo web app for (WebSocket-based) lattice-core
  * [medley](https://github.com/jwoertink/medley) - A mixture of music related methods
- * [os-crystal](https://github.com/lbguilherme/os-crystal) - x86 Kernel implemented in Crystal
```

Comment:
That's my only disagreement, I think it's an awesome proof of concept!

Reply:
Maybe. But it is just a proof of concept and not going to be improved/updated. So sooner or later it will not compile and we will need to remove it.

Reply:
And it may not even work with latest Crystal. I am not able to build it :/

Reply:
You have a point, but, as I see, this list has the purpose to show Crystal users, and people interested in Crystal, great projects and what the language can do. I think an implementation of a kernel using the language shows for newcomers its power.

Reply:
I would be glad to leave in the list once we would be sure it still works. I think broken packages will not be helpful for newcomers.

Reply:
I have opened a discussion https://github.com/lbguilherme/os-crystal/issues/3

Reply:
@veelenga I tried it now and I'm able to compile and run it on Ubuntu. Maybe there are some OS specific link flags in Makefile?